HealthcareSystem/Backend/Service/DrugAndTherapy/DrugService.cs

```diff
@@ -144,11 +144,24 @@ public List<Drug> GetDrugsByRoomNumber(int roomNumber)
             return confirmedDrugsInRoom;
         }
 
-        public DrugInRoom GetDrugInRoomByDrugId(int drugId)
+        public List<Drug> GetDrugWithRoomForSearchTerm(string searchTerm)
         {
-            DrugInRoom drugInRoom = _drugInRoomRepository.GetDrugInRoomByDrugId(drugId);
+            List<Drug> drugs = ViewConfirmedDrugs();
+            List<Drug> validEDrugs = new List<Drug>();
```

Comment:
Why validEDrugs? Does the E mean something?

Reply:
it was there by mstake